content/labs/intro-to-ada/chapters/standard_library_numerics.rst

```diff
@@ -401,6 +401,8 @@ circle after rotation.
                 Point #4:  -90.00 degrees (= 270 degrees)
                 Point #5:    0.00 degrees
 
+		#. To rotate a complex number simply multiply it by a unit vector whose arg is the radian angle to be rotated: :math:`Z = e^\frac{2 \pi}{N}`
```

Comment:
This doesn't display properly, and I couldn't figure out why. Looking for some help :)

Reply:
```suggestion
            #. To rotate a complex number simply multiply it by a unit vector
               whose arg is the radian angle to be rotated:
               :math:`Z = e^\frac{2 \pi}{N}`
```

Reply:
The paragraph with the formula was aligned with the previous code block...

Reply:
In any case, I'm not totally convinced that the formula is actually useful here. Isn't the previous explanation (in simpler terms) enough?

Reply:
I think it depends on the user's knowledge of complex numbers; if we assume the reader remembers complex number operations then this would be unnecessary.

But the fact that we are describing the relationship between the coordinate form and the polar form suggests we assume no experience with complex numbers. 

In this case the reader understands that they want to rotate the vector through those angles, but they might not know how to do it. So thats what the 3rd point is meant to give them.

Reply:
Ok, fine with me. :-)